ansible/roles/test/files/tools/loganalyzer/loganalyzer_init.yml

```diff
@@ -55,6 +55,14 @@
 - name: create output directory for current test run
   file: path="{{ test_out_dir }}" state=directory
 
+- name: backup logrotate cron configuration
+  shell: cp /etc/cron.d/logrotate /tmp
+  become: yes
+
+- name: Disable logrotate during test run
+  shell: sed -i 's/^/#/g' /etc/cron.d/logrotate
+  become: yes
+
```

Comment:
I feel like we should also check to make sure no logrotate processes are running before proceeding. Even though you disable the cron job, there's a possibility the previous cron job is still running.

Reply:
Good point. I updated the logic to wait 1min before proceed as cron checks configuration update every 1min. https://www.freebsd.org/cgi/man.cgi?cron

Reply:
Cron will definitely pick up the config change before the next run, as I have scheduled logrotate to run every minute, therefore, the next time cron wakes up, it will reload its config and will not kick off logrotate.

My concern is not when cron will pick up the configuration change, but rather that we need to wait for any already-running instances of logrotate from previous cron jobs to exit before starting another process.